akka-docs/_sphinx/exts/includecode2.py

```diff
@@ -43,7 +43,7 @@ def run(self):
         encoding = self.options.get('encoding', env.config.source_encoding)
         codec_info = codecs.lookup(encoding)
         try:
-            f = codecs.StreamReaderWriter(open(fn, 'U'),
+            f = codecs.StreamReaderWriter(open(fn, 'Ub'),
```

Comment:
Yes. The added `b` fixes a problem when Python 3 is used when generating the docs.